### ISSUE TITLE
Design debt: look type and function on on-page tabs

### ISSUE THREAD

**Issue by connorscarolyns** [cds-snc/platform-forms-client#4727]
There are a couple of different styles of on-page tabs. These should be brought into alignment and restyled to fit the current look and feel.

Fable testing also showed that tabs are not functioning as tabs at the semantic level and they should be reworked to meet WCAG standards.

**Comment by anikbrazeau** [comment]
Related to / duplicate of #3006
